PeerioTechnologies/peerio-icebear
# chore: in Onboarding, 'add avatar' and 'save key' text was switched

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/9405/desktop-switch-bonus-strings
 
#### Testing instructions  
in Onboarding, the subtitles below "Upload your Avatar" and "Save your Account Key..." should now display the correct text

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
